src/Equinor.ProCoSys.IPO.MainApi/MainApi/McPkg/MainApiMcPkgService.cs

```diff
@@ -83,21 +84,23 @@ public class MainApiMcPkgService : IMcPkgApiService
 
         public async Task ClearM01DatesAsync(
             string plant,
-            int invitationId,
             string projectName,
             IList<string> mcPkgNos,
-            IList<string> commPkgNos)
+            IList<string> commPkgNos,
+            int? invitationId)
         {
             var url = $"{_baseAddress}McPkgs/ClearM01" +
                       $"?plantId={plant}" +
                       $"&api-version={_apiVersion}";
-            var bodyPayload = new
+            dynamic bodyPayload = new ExpandoObject();
```

Comment:
I have never used this, but you have tested that it works with both clearing and not clearing external reference?

Reply:
I guess it will be the same result, but I don't understand why dynamic bodyPayload = new ExpandoObject(); + setting props one by one is better/easier to read. I find old code with 
var bodyPayload = new
{
 // init props
}
better